content/source/docs/enterprise/api/admin/settings.html.md

```diff
@@ -0,0 +1,492 @@
+  --request GET \
+  https://app.terraform.io/api/v2/admin/settings/saml
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "saml",
+    "type": "settings/saml",
+    "attributes": {
+      "enabled": true,
+      "idp-cert": "SAMPLE-CERTIFICATE",
+      "slo-target": "https://example.com/slo",
+      "sso-target": "https://example.com/sso",
+      "attr-groups": "MemberOf",
+      "attr-site-admin": "SiteAdmin",
+      "site-admin-role": "site-admins",
+      "sso-api-token-session-timeout": 1209600
+    }
+  }
+}
+```
+
+## Update SAML Settings
+`PATCH /api/v2/admin/settings/saml`
+
+-> **Note:** This endpoint cannot be accessed with [organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts). You must access it with a [user token](../users-teams-organizations/users.html#api-tokens) which has the `site-admins` permission.
+
+Status  | Response                                         | Reason
+--------|--------------------------------------------------|-------
+[200][] | [JSON API document][] (`type: "settings/saml"`) | Successfully updated SAML settings
+[404][] | [JSON API error object][]                    | User unauthorized to perform action
+[422][] | [JSON API error object][]                    | Malformed request body (missing attributes, wrong types, etc.)
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+[JSON API document]: https://www.terraform.io/docs/enterprise/api/index.html#json-api-documents
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+### Request Body
+
+This PATCH endpoint requires a JSON object with the following properties as a request payload.
+
+Properties without a default value are required if `data.attributes.enabled` is set to `true`.
+
+Key path                    | Type   | Default | Description
+----------------------------|--------|---------|------------
+`data.attributes.enabled`    | bool   | `false` | Allows SAML to be used. If true, all remaining attributes are required.
+`data.attributes.idp-cert`   | string |         | Identity Provider Certificate specifies the PEM encoded X.509 Certificate as provided by the IdP configuration.
+`data.attributes.slo-target` | string |         | Single Log Out URL specifies the HTTPS endpoint on your IdP for single logout requests. This value is provided by your IdP configuration.
+`data.attributes.sso-target` | string |         | Single Sign On URL specifies the HTTPS endpoint on your IdP for single sign-on requests. This value is provided by your IdP configuration.
+`data.attributes.attr-groups`| string | `"MemberOf"` | Team Attribute Name specifies the name of the SAML attribute that determines team membership. The value of this attribute in the SAML assertion must be a string containing a comma-separated list of team names.
```

Comment:
I don't think the second sentence is correct. The value can also list each team name in a separate item. Also it doesn't seem relevant here, so maybe just remove it or link to the documentation on it?